examples/gender.rs

```diff
@@ -0,0 +1,47 @@
+use message_format::ast::*;
+
+pub fn main() {
+    // Anne published a post in Birthday Party
```

Comment:
Suggestion: add the generic form, as represented by the code:
`{userName} published a post in {groupName}`
(or, if we don't want it ICU, `{$userName} published a post in {$groupName}`)